FAQ.md

```diff
@@ -44,19 +44,8 @@ ilp-plugin-xrp-asym-client Error creating the payment channel: tecUNFUNDED One o
 
 ### Can I run multiple instances of Codius from one wallet?
 Yes! 
-* Add a unique name to your moneyd configuration using the below function:
```

Comment:
I'd keep this section for anyone not running `moneyd-uplink-xrp` on version 1.1.2, perhaps following the the statement that 1.1.2 already assigns a name. I would also suggest a bullet point on how to check the version of `moneyd-uplink-xrp`.